angular.json

```diff
@@ -101,7 +101,8 @@
             "karmaConfig": "components/karma.conf.js",
             "polyfills": "components/polyfills.ts",
             "tsConfig": "components/tsconfig.spec.json",
-            "scripts": []
```

Comment:
Yes, the upgrade of angular_devkit/build-angular from 0.6 to 0.8 drops code coverage since the `sourceMap` option is not enabled. They fixed it in the master but reluctantly cherry-picked to the 6.x branch. See https://github.com/angular/angular-cli/pull/12291#issuecomment-429271768